cluster/examples/kubernetes/ceph/csi/template/rbd/csi-rbdplugin-provisioner.yaml

```diff
@@ -100,6 +100,26 @@ spec:
               mountPath: /etc/ceph-csi-config/
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
+        - name: liveness-prometheus
```

Comment:
there is a dummy headless service already created for cephfs and rbd please make use of it https://github.com/rook/rook/blob/master/pkg/operator/k8sutil/statefulset.go#L30, 
@travisn can we make changes in headless service creation function?

Reply:
@Madhu-1 How do you see that would be used? The csi templates are used to create the resources for the csi driver, instead of calling those helpers.

Reply:
currently, we are creating a [dummy service](https://github.com/rook/rook/blob/master/pkg/operator/k8sutil/statefulset.go#L39)  for [statefulset](https://github.com/rook/rook/blob/master/pkg/operator/ceph/csi/spec.go#L170) .The `1234` service port won't be used anywhere. what I was thinking was just update `1234` to proper service port so that liveness can use.

Reply:
@travisn can this be addressed later also?
note: without suggested change, there will be 2 services created for rbd and cephfs provisioner.

Reply:
I see four services defined in the template yaml in this PR, two for rbd and two for cephfs. They each have different ports and label selectors. Are you saying a single dummy service can be created in place of those four?